docs/fundamentals/jcloud/advanced.md

```diff
@@ -73,6 +73,47 @@ executors:
           type: efs
 ```
 
+### Gateway
+
+For the internet exposure, JCloud now provides support for [Kong gateway](https://konghq.com/products/api-gateway-platform). 
+
+We currently support 2 types of gateway.
```

Comment:
I don't think we should say that we support something *now*, this documentation might remain untouched for a while, so I think it is better to use timeless phrasing: I suggest the following:
```suggestion
We currently support two types of gateway for internet exposure: [Kong gateway](https://konghq.com/products/api-gateway-platform) and [Application Load Balancer (ALB)](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/introduction.html).
```